web/src/main/webapp/xslt/ui-metadata/form-builder.xsl

```diff
@@ -1549,7 +1549,27 @@
   -->
   <xsl:template mode="render-for-field-for-attribute"
                 match="gn:attribute[not(@name = ('ref', 'parent', 'id', 'uuid', 'type', 'uuidref',
-    'xlink:show', 'xlink:actuate', 'xlink:arcrole', 'xlink:role', 'xlink:title', 'xlink:href'))]"
+    'xlink:show', 'xlink:actuate', 'xlink:arcrole', 'xlink:role', 'xlink:title', 'xlink:href', 'xlink:type'))]"
+                priority="4">
+    <xsl:param name="ref"/>
+    <xsl:param name="insertRef" select="''"/>
+
+    <xsl:variable name="attributeLabel" select="gn-fn-metadata:getLabel($schema, @name, $labels)"/>
+    <xsl:variable name="fieldName"
+                  select="concat(replace(@name, ':', 'COLON'), '_', $insertRef)"/>
+    <button type="button" class="btn btn-default btn-xs btn-attr btn-nil btn-xs gn-attr-{replace(@name, ':', '_')} {if ($isDisplayingAttributes = true()) then '' else 'hidden'}"
```

Comment:
Why not using existing `gn-attr-gco_nilReason` class ?

![image](https://user-images.githubusercontent.com/1701393/114559726-51efc080-9c6c-11eb-8193-bda968f2d7fe.png)


So as said by Jose, you can probably keep only one template.

Reply:
Maybe the change should only be in simple view to not show the nilReason unless we display all attributes ?

![image](https://user-images.githubusercontent.com/1701393/114560575-1bff0c00-9c6d-11eb-81aa-f77726efcd0e.png)

Reply:
I think it's best for me to go back to the drawing board to think of a solution that fits all.

As a side note, it clearly shows I'm no `xsl` expert